src/js/correlation/components/ZeekCorrelation.tsx

```diff
@@ -1,27 +1,25 @@
-import {connect} from "react-redux"
 import React from "react"
 import classNames from "classnames"
 import * as d3 from "d3"
 import isEqual from "lodash/isEqual"
+import {zng} from "../../../../zealot/dist"
+import brim from "../../brim"
+import {createZeekLog} from "../../brim/zeekLog"
+import {submitSearch} from "../../flows/submitSearch/mod"
+import {viewLogDetail} from "../../flows/viewLogDetail"
+import {UID_CORRELATION_LIMIT} from "../../searches/programs"
+import SearchBar from "../../state/SearchBar"
+import {BrimEvent} from "../models/BrimEvent"
+import {sort} from "../util/sort"
+import {useDispatch} from "react-redux"
 
-import {DispatchProps} from "../state/types"
-import {UID_CORRELATION_LIMIT} from "../searches/programs"
-import {submitSearch} from "../flows/submitSearch/mod"
-import {viewLogDetail} from "../flows/viewLogDetail"
-import SearchBar from "../state/SearchBar"
-import brim from "../brim"
-import dispatchToProps from "../lib/dispatchToProps"
-import {zng} from "zealot"
-import {createZeekLog} from "../brim/zeekLog"
-
-type OwnProps = {
+type Props = {
   logs: zng.Record[]
   log: zng.Record
 }
 
-type Props = DispatchProps & OwnProps
-
-export default function UidTimeline({logs, log, dispatch}: Props) {
+export default function ZeekCorrelation({logs, log}: Props) {
```

Comment:
I've moved the component for the UID Correlation into the `src/js/correlation` folder. I've also moved all supporting components/classes/hooks into this folder. This folder will be a ppl folder soon as well.